app/src/main/java/kth/jjve/xfran/SaveResultsActivity.java

```diff
@@ -0,0 +1,157 @@
+package kth.jjve.xfran;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.android.material.navigation.NavigationView;
+
+import java.util.ArrayList;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.view.GravityCompat;
+import androidx.drawerlayout.widget.DrawerLayout;
+import kth.jjve.xfran.models.Workout;
+
+import static android.view.View.GONE;
+
+public class SaveResultsActivity extends AppCompatActivity implements NavigationView.OnNavigationItemSelectedListener {
+
+    private final String LOG_TAG = getClass().getSimpleName();
+
+    /*_________ INTENT _________*/
+    private int position;
+    private Workout mWorkout;
+
+    /*_________ VIEW _________*/
+    private DrawerLayout drawerLayout;
+    private NavigationView navigationView;
+    private Toolbar toolbar;
+
+    /*------ HOOKS ------*/
+    private Button saveButton;
+    private Button cancelButton;
+    private TextView mName;
+    private TextView mDescription;
+    private TextView mExercises;
+    private Button mSaveButton;
+    private Button mPlanButton;
```

Comment:
All the buttons can be declared on the same line, so:
`private Button saveButton, cancelButton, mSaveButton, mPlanButton`
Also: having saveButton and mSaveButton is a bit confusing, consider renaming one of the two :)

Reply:
same holds for the textviews btw :)